src/background.js

```diff
@@ -68,5 +73,8 @@ app.on('ready', async () => {
     // Install Vue Devtools
     await installVueDevtools()
   }
-  mainWindow = createMainWindow()
+  mainWindow = createMainWindow(app.getLocale())
+
+  // Uncomment this to show console.
+  // mainWindow.webContents.openDevTools()
```

Comment:
@jayperryworks - if you need to debug things, uncomment this to turn on debugging.